src/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -283,6 +298,23 @@ object SaveToRedis {
     arr.foreach(pipeline.rpush(listName, _))
     pipeline.sync
   }
+  /**
+   * @param host addr and port of a target host
+   * @param listName
+   * @param listSize
+   * @param arr values which should be saved in the target host
+   * save all the values to listName(list type) to the target host
+   */
+  def setFixedList(host: (String, Int), listName: String, listSize: Int, arr: Iterator[String]) = {
+    val jedis = new Jedis(host._1, host._2)
+    val pipeline = jedis.pipelined
+    arr.foreach(pipeline.lpush(listName, _))
+    if (listSize > 0) {
+      pipeline.ltrim(listname, 0, listSize - 1)
```

Comment:
listname -> listName
sorry, I didn't notice it too. :-)

Could you please merge all the commits into a single one? :+1: